### ISSUE TITLE
ValueError when loading from pickled scdata

### ISSUE THREAD

**Issue by Colelyman** [ManuSetty/wishbone#27]
I get an error whenever I try to load in a pickled `scdata` object. I use the functions outlined in the docs to both save and load the pickled object: 
```python
scdata.save('mouse_marrow_scdata.p') 
scdata = wishbone.wb.SCdata.load('mouse_marrow_scdata.p')
```
This is the output of the attempted load. 
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-4-de71f0930d1e> in <module>()
----> 1 scdata = wishbone.wb.SCData.load('./sandberg.p')

/home/colelyman/anaconda3/envs/wishbone/lib/python3.5/site-packages/wishbone/wb.py in load(cls, fin)
    125         with open(fin, 'rb') as f:
    126             data = pickle.load(f)
--> 127         scdata = cls(data['_data'], data['_metadata'])
    128         del data['_data']
    129         del data['_metadata']

/home/colelyman/anaconda3/envs/wishbone/lib/python3.5/site-packages/wishbone/wb.py in __init__(self, data, data_type, metadata)
     86         if not (isinstance(data, pd.DataFrame)):
     87             raise TypeError('data must be of type or DataFrame')
---> 88         if not data_type in ['sc-seq', 'masscyt']:
     89             raise RuntimeError('data_type must be either sc-seq or masscyt')
     90         if metadata is None:

/home/colelyman/anaconda3/envs/wishbone/lib/python3.5/site-packages/pandas/core/generic.py in __nonzero__(self)
    890         raise ValueError("The truth value of a {0} is ambiguous. "
    891                          "Use a.empty, a.bool(), a.item(), a.any() or a.all()."
--> 892                          .format(self.__class__.__name__))
    893 
    894     __bool__ = __nonzero__

ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().
```

I am willing to perform the fix myself and submit a pull request, but I wanted to know if you had any ideas.

Thanks!

**Comment by ManuSetty** [comment]
Thanks for bringing this to notice! A parameter was missing in the class instantiation when loading from the pickle file. This has been fixed in 724088e6a6a45bd3e642cb70a9ebf85f6b2f645e and pushed to the [develop](https://github.com/ManuSetty/wishbone/tree/develop) branch. I will include this in the next release. Please pull from the develop branch.

**Comment by Colelyman** [comment]
Great, thank you!
